src/test/ui/async-await/track-caller/issue-104588-no-op-panic-track-caller.rs

```diff
@@ -0,0 +1,78 @@
+// run-pass
```

Comment:
Is it possible to use `// revisions` to test this with and without the feature gate? Or is there any code that's exercising this feature with `closure_track_caller` enabled?

Otherwise I'm not sure I understand what this code is supposed to be testing :thinking:

Reply:
it's a copy paste of that other test with a couple lines changed

and the other tiny test i added just tests compilation so this one tests that the runtime behavior is as we expect.

but maybe this test isn't needed?

Reply:
> it's a copy paste of that other test with a couple lines changed

Depending on what lines you changed, you may be able to use revisions instead: https://rustc-dev-guide.rust-lang.org/tests/compiletest.html#revisions

Reply:
I got it to work with revisions now instead of the separate file